MAINTAINERS.md

```diff
@@ -14,3 +17,15 @@
 
 ### Updating Practices
 To ensure common practices as maintainers, all practices are expected to be documented here or enforced through github actions.  There should be no expectations beyond what is documented in the repo [CONTRIBUTING.md](./CONTRIBUTING.md) and OpenSearch-Project [CONTRIBUTING.md](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md).  To modify an existing processes or create a new one, make a pull request on this MAINTAINERS.md for review and merge it after all maintainers approve of it.
+
+# Practices
+
+## Reverting Commits
```

Comment:
> Are we make reverting commits a part of regular basis practices?

@cliu123 Could you include the language that you'd like to see changed or your recommendation you have?

Reply:
The language looks good to me. My question is on the process of reverting and how we can minimize revert.

Reply:
If the to-be-reverted change is blocking, failing builds or disrupting in other ways, and a fix forward is not quick, we should revert and work on a fix that reverts the revert plus any other changes needed to fix the initial issue. @cliu123 I believe that's what you are pointing at, as in reverting should be a last resort to unblock rather than the go-to strategy for fixing things, and I think that's exactly what that first paragraph on this PR conveys... but if it's not clear please feel free to suggest re-wording.